### DISCUSSION TITLE
CMake compilation error on win10 system?

### DISCUSSION THREAD

**Discussion by cuixing158** [apache/incubator-mxnet#19331] [Q&A] [answered]
![cmakeError](https://user-images.githubusercontent.com/22857777/95669733-37059c00-0bb6-11eb-8421-148a7b7aed58.png)
The environment is configured, and how to solve this error, do I have to compile the 3rdparty folder? Empty files inside

**Comment by leezu** [comment, marked as answer]
You need to initialize the git submodules. `git submodule update --init --recursive`
